### ISSUE TITLE
.ts video format Support

### ISSUE THREAD

**Issue by mg8x** [bipinkrish/file-converter-telegram-bot#18]
Could you please add .ts video format. It would be great!!

**Comment by bipinkrish** [comment]
Ffmpeg supports all, it just needs to be the array of supported formats

**Comment by mg8x** [comment]
Where can i edit arrays?

**Comment by bipinkrish** [comment]
Can you handle python? because `.ts` is not used mostly, and my code is a mess to even extend

**Comment by mg8x** [comment]
Yes i can handle python. But know i have exams so not until 30 days :(
